src/octopeer-github/content/ElementSelectionBehaviour/NameElementSelectionBehaviour/OtherContributerNameElementSelectionBehaviour.ts

```diff
@@ -11,7 +11,7 @@ class OtherContributerNameElementSelectionBehaviour extends NameElementSelection
      */
     public constructor(database: DatabaseAdaptable) {
         super(database);
-        this.elementDsc = "[placeholder]";
+        this.elementDsc = "teststss";
```

Comment:
Yes! I had to change the "[placeholder]" selector, since this returns all elements with a placeholder attribute (which is every textarea)